src/ExtractGQL.ts

```diff
@@ -210,7 +212,8 @@ export class ExtractGQL {
     var operationNames = new Set<string>();
     var duplicates = new Set<string>();
     for (var i = 0; i < doc.definitions.length; i++) {
-      const name = doc.definitions[i].name.value;
+      console.log(doc.definitions[i]);
```

Comment:
I left this in by accident and this was causing the output to include a bunch of stuff it wasn't supposed to.